### ISSUE TITLE
Glyph Ambiguity 

### ISSUE THREAD

**Issue by ghost** [YuurinBee/SAIBA-45#3]
I think this font is awesome, and I wanted to suggest some small improvements.

One thing I noticed while using this font is that I find "2" and "3" to be hard to read. For some reason 2 appears like a 3 to me, and 3 doesn't really look like anything. Maybe it's just my brain.

While I was at it I took a look at some of the other glyphs, "U" and "V" look somewhat ambiguous, "X" looks kinda like an X in context, but not by itself, and "J" looks like a Q.

I think it should be possible to slightly change the styling to make the glyphs more readable while still looking cool.

**Comment by ghost** [comment]
Here's a rough mockup I made in GIMP:
![image](https://github.com/YuurinBee/SAIBA-45/assets/4300747/b2971f26-42b8-48ca-be62-db2678192dca)

**Comment by YuurinBee** [comment]
> I think this font is awesome, and I wanted to suggest some small improvements.

Firstly, thanks for taking the time to actually visit the GitHub repo and share your feedback and creative input. It is always appreciated for the true nature of open-source development.

> One thing I noticed while using this font is that I find "2" and "3" to be hard to read. For some reason 2 appears like a 3 to me, and 3 doesn't really look like anything. Maybe it's just my brain.

I agree with you on this. There are some glyphs and characters that I did not feel were "perfect" just yet, but when you work on projects for so long (this one started in 2021), you will not progress unless you finalize and publish your current iterations. So, I'm happy that you have put under the microscope again feelings and thoughts that I've also had, but haven't revisited to put my finger on exactly.
 
> While I was at it I took a look at some of the other glyphs, "U" and "V" look somewhat ambiguous, "X" looks kinda like an X in context, but not by itself, and "J" looks like a Q.

Funny enough, I did exactly this for the "**J**" already in the Cyrillic Alphabet for `v1.1` (already released). 
![image](https://github.com/YuurinBee/SAIBA-45/assets/32799031/a9970a6b-9a16-459d-8936-d328bb90a360)

Once again, your comments are also chirping in on similar feelings I've had. I noticed that some letters (standalone) look really cool and unique, but when combined they lose some legibility and it shows there is a lot of room for improvement for the overall font family. 

> I think it should be possible to slightly change the styling to make the glyphs more readable while still looking cool.

Thanks for your feedback @JoshuaFern, I agree with 100% of what you've shared, as I mentioned I also had similar thoughts throughout the process. I intend to credit you for your relative and helpful input on this open-source font family and will add you to the Contributors on the `README.md`.

**Comment by ghost** [comment]
Thanks for your response, I enjoyed reading it.

Maybe I should start a new issue for this, but I also wanted to ask you about licensing. I noticed the font is currently licensed under GPL v3 and I'm generally a fan of this license, but I believe it's designed for licensing code and not fonts. Have you looked at using something like the Open Font License instead?

**Comment by YuurinBee** [comment]
@JoshuaFern I actually have the least bit of experience with legalities and licensing, there are so many different types and the main or ideal licensing I wanted is that people could literally use it for whatever, though I wouldn't want people selling it and making money off it by doing nothing or adding nothing, but that's the risk we take with open-source haha. Please feel free to open a new issue and I am happy to research and look into the most ideal licensing that is suitable, especially if you have suggestions.

**Comment by YuurinBee** [comment]
Thanks for all your input and contributions @JoshuaFern - I will have to modify the glyphs and then probably release Version 1.2 in the near future here. If anything else comes up, feel free to share.

**Comment by YuurinBee** [comment]
![image](https://github.com/YuurinBee/SAIBA-45/assets/32799031/614cb441-15cc-47d6-8b65-a5d9f94387c2)


Greetings! Finally, getting back around to enhancing the font and reducing ambiguity without compromising the style. In the spirit of open-source culture, I wanted to share the latest iterations and get feedback from **SAIBA-45** contributors @JoshuaFern @JSKitty. Check out the sample above and share your honest feedback (it is not necessarily final, just new iterations). I am testing both legibility with the characters/letters in words and also individually (this is where the biggest challenge resides).

`SAIBA-45 Font Family now has over 5,000 downloads!`

---

**J**
> This actually should've been updated already. The English/Latin Alphabet was inconsistent with the Cyrillic Alphabet I had already created. Good catch and suggestion Joshua.

**U**
> For now, this is certainly more legible, but I feel it lacks some of the SAIBA-45 style and character. I took away the big 45 degree cut into the bottom left side, which I feel resembles more of a "V" than a "U". I played around with a handful of variations and felt like it kept taking away from the foundation of the shape, so I stuck with simple and minor stylistic changes.

**V**
> The V could possible use a little bit more work, but that strong 45 degree cut in the bottom left (that was removed from the U) certain makes the left part of the glyph look more like a V. 

**X**
> I am quite happy with the way the X turned out and can already picture this a unique logo for a handful of different types of ideas and projects. At first, I wasn't sure if it resembled an "H", but not really with the diagonal line. The "H" also has a hidden little easter egg that it keeps the shape of a lowercase "h", inside the big "H".

**2**
> This one just became incredible simplified, but certainly easier to read. It is a challenge to take away parts without removing the foundation of glyph, reducing legibility, but also keeping the abstract style. Can easily read this far away and small in size. 

**3**
> I couldn't stand keeping it too simple and just adding the right column, so I maintained the 45 degree angle in the bottom right and also the legibility.

---

**Comment by ghost** [comment]
I'm happy with the way J, X, and 3 came out.

For 2 and U, I feel like a bit more style would be welcomed, but it's certainly not ambiguous to me.

For V, I still find ambiguity in this glyph. It's not immediately clear to me what letter it is.

I spoke to a friend on Discord and sent them this picture and had them read the letters outloud. They thought the X was an N at first, and wasn't sure what the V was. After explaining that the X was an X they agreed it looked like an X.

**Comment by YuurinBee** [comment]
Agreed once again with your comments. I also want to play more with the "U" and "2" to make them more consistent with the SAIBA-45 style, but not losing legibility. Regardless, finally back on the drawing board and Version 1.2 is underway!

> "I spoke to a friend on Discord and sent them this picture and had them read the letters outloud. They thought the X was an N at first, and wasn't sure what the V was. After explaining that the X was an X they agreed it looked like an X."

Oof. Yeah, it's always important to do testing in all types of design and get feedback from really as many sources as possible (as long as you can manage the direction and not lose focus). The "N", not seeing that at all, but the human mind is miraculous haha. I will note this and also share with some others to get some feedback. I feel like if I put this on a mech model like a mobile suit or vehicle, it would be pretty clear... or "Model X", there wouldn't be much room for speculation.

**Comment by ghost** [comment]
>  The "N", not seeing that at all, but the human mind is miraculous haha.

After looking at it again, I can clearly see the N now. It's like one of those optical illusions where you can see the woman rotating counter clockwise or clockwise. It requires a shift in perception.

**Comment by YuurinBee** [comment]
> After looking at it again, I can clearly see the N now. It's like one of those optical illusions where you can see the woman rotating counter clockwise or clockwise. It requires a shift in perception.

I also see this now haha. Interesting, well this might be a regression. I also got some feedback and people literally saw both. Unfortunately, the first "X" is clearly an X in comparison. Back to the drawing board then. Gotta love human psychology and perception. Thanks for the valuable feedback @JoshuaFern!

**Comment by mglsj** [comment]
> Agreed once again with your comments. I also want to play more with the "U" and "2" to make them more consistent with the SAIBA-45 style, but not losing legibility. Regardless, finally back on the drawing board and Version 1.2 is underway!
> 
> > "I spoke to a friend on Discord and sent them this picture and had them read the letters outloud. They thought the X was an N at first, and wasn't sure what the V was. After explaining that the X was an X they agreed it looked like an X."
> 
> Oof. Yeah, it's always important to do testing in all types of design and get feedback from really as many sources as possible (as long as you can manage the direction and not lose focus). The "N", not seeing that at all, but the human mind is miraculous haha. I will note this and also share with some others to get some feedback. I feel like if I put this on a mech model like a mobile suit or vehicle, it would be pretty clear... or "Model X", there wouldn't be much room for speculation.

Any update on this?
